content/en/monitors/monitor_types/forecasts.md

```diff
@@ -15,91 +15,101 @@ further_reading:
+* Trigger an alert when the edge of the forecast confidence bounds goes `above` or `below`
+* the threshold within the next `24 hours`, `1 week`, `1 month`, etc.
+* Alert threshold: >= `<NUMBER>`
+* Alert [recovery threshold][3]: < `<NUMBER>`
 
-Datadog automatically sets the **Advanced** options for you by analyzing your metric. Note that any changes in the **Define the metric** section could change the advanced options.
+#### Advanced Options
 
-* You can change the forecasting algorithm to be used here. See the [Forecast algorithms](#forecast-algorithms) section for tips on how to choose the best algorithm for your use case. Each algorithm also has additional settings is described in the next section.
-* Datadog recommends using larger intervals between points to avoid having noise influence the forecast too much.
-* The number of deviations controls the width of the range of forecasted values. A value of 1 or 2 should be large enough to forecast most "normal" points accurately.
+Datadog automatically analyzes your chosen metric and sets several parameters for you. However, the options are available to edit under **Advanced Options (auto)**:
 
-Then, you can choose if the monitor requires a full window of data for evaluation. Setting it to "Requires" forces the monitor to ignore (show "No Data" as the monitor state) any series that doesn't have data going back to the start of the time range shown in the Evaluation Window graph.
+{{< img src="monitors/monitor_types/forecasts/advanced_options.png" alt="Advanced options" responsive="true" style="width:80%;">}}
 
-Complete all steps in the New Monitor form (**Say what's happening**, etc.) and click **Save** to create the Forecast monitor.
+| Option                     | Description                                                                                                             |
+|----------------------------|-------------------------------------------------------------------------------------------------------------------------|
+| [Algorithm](#algorithms)   | The forecast algorithm (`linear` or `seasonal`)                                                                         |
+| Model                      | The forecast model (`default`, `simple`, or `reactive`) for the linear algorithm                                        |
+| Seasonality                | The forecast seasonality (`hourly`, `daily`, `weekly`, or `monthly`) for the seasonal algorithm                         |
+| [Daylight&nbsp;savings][4] | Available for `seasonal` forecast monitors with `daily`, `weekly`, or `monthly` seasonality.                            |
+| [Rollup][5]                | The rollup interval&mdash;larger intervals between points avoid noise influence on the forecast.                        |
+| Deviations                 | The width of the range of forecasted values&mdash;a value of 1 or 2 is generally large enough for most "normal" points. |
```

Comment:
The "acceptable variation" instead of "width"?  I'm actually thinking it would be helpful to get a review from data-science here as well since they really own the evaluation  of these monitors.

Reply:
`width` is used in the current docs.

Reply:
Ok that definitely makes sense 👍